packages/webpack-config/src/helpers/paths.js

```diff
@@ -117,10 +117,20 @@ export function join(...paths) {
 export const useHtmlCompiler = Boolean(config.hbsEntry && config.hbsTarget);
 export const hbsEntry = useHtmlCompiler ? resolveApp(config.hbsEntry) : "/";
 export const hbsTarget = useHtmlCompiler ? resolveApp(config.hbsTarget) : "/";
-// check if a hbs partial dir is provided
-export const hbsPartialDir = {
-  partialDirs: config.hbsPartialDir ? [resolveApp(config.hbsPartialDir)] : []
-};
+
+/******************************************************************************
+ ** handlerbars-loader options
+ ******************************************************************************/
+const handlebarsLoaderOptions = config.handlebarsLoaderOptions || {};
```

Comment:
it would consitent to define defaults ini `src/config/default.config.js`

Reply:
what would you suggest as default values?

Reply:
{} ?

Reply:
done